test/common/report.js

```diff
@@ -25,7 +25,12 @@ function findReports(pid, dir) {
 }
 
 function validate(filepath) {
-  validateContent(JSON.parse(fs.readFileSync(filepath, 'utf8')));
+  const report = fs.readFileSync(filepath, 'utf8');
+  if (process.execArgv.includes('--report-compact')) {
```

Comment:
Should this line check for `process.report.compact` in case the test is not started with the CLI flag?